packager/mpd/test/xml_compare.cc

```diff
@@ -80,10 +78,17 @@ bool CompareContents(xmlNodePtr node1, xmlNodePtr node2) {
       reinterpret_cast<const char*>(node1_content_ptr.get());
   std::string node2_content =
       reinterpret_cast<const char*>(node2_content_ptr.get());
-  base::ReplaceChars(node1_content, "\n", "", &node1_content);
-  base::TrimString(node1_content, " ", &node1_content);
-  base::ReplaceChars(node2_content, "\n", "", &node2_content);
-  base::TrimString(node2_content, " ", &node2_content);
+
+  node1_content.erase(
+      std::remove(node1_content.begin(), node1_content.end(), '\n'),
+      node1_content.end());
+  node2_content.erase(
+      std::remove(node2_content.begin(), node2_content.end(), '\n'),
+      node2_content.end());
+
+  node1_content = absl::StripAsciiWhitespace(node1_content);
```

Comment:
I think `base::TrimString` and `absl::StripAsciiWhitespace` are equivalent but I'm not sure

Reply:
I expect so, yes